Dockerfile.ubuntu

```diff
@@ -109,21 +110,19 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
             nlohmann-json3-dev \
             python2 \
             python2-dev \
-            python-setuptools \
-            python3 \
-            python3-pip \
-            python3-dev \
-            python-is-python3 \
-            python3-setuptools \
-            python3-virtualenv \
-            python3-numpy \
-            python-is-python3 \
             unzip \
             vim \
             xz-utils && \
             apt-get clean && \
             rm -rf /var/lib/apt/lists/*
 
+RUN apt-get update && apt-get install -y software-properties-common gpg gpg-agent --no-install-recommends && \
+    add-apt-repository ppa:deadsnakes/ppa && \
```

Comment:
Is there need to adding repository for ubuntu 20.04 ?

Reply:
it is not needed for ubuntu20 and is not used. I was not adding a condition for the code simplicity